content/docs/start/model-management/model-registry.md

```diff
@@ -0,0 +1,289 @@
+parameters are descriptive and optional and will also show up in the model
+registry.
+
+Now we just need to run the python script which includes this code to cache and
+register the model.
+
+If you are following our example repository then the training script is included
+in a dvc pipeline [we prepared](/doc/start/experiments/experiment-pipelines) in
+the Experiment Management guide, so there we just call `dvc exp run` to run the
+entire experiment pipeline.
+
+We now commit the result to git (and push it to our git remote) and the new
+model will show up in the model registry in Studio. We will also push the
+resulting artifact from DVC cache to our DVC Remote by calling `dvc push`.
+
+<details id="push-click-to-see-how-artifacts-are-registered">
+
+#### 💡 Expand to see how adding models to the registry works under the hood
+
+When we call the `log_artifact()` method, DVC takes all the information we
+provided in the call and edits the `dvc.yaml` file which will now contain the
+following lines
+
+```yaml
+artifacts:
+  pool-segmentation:
+    path: models/model.pkl
+    type: model
+    desc:
+      This is a Computer Vision (CV) model that's segmenting out swimming pools
+      from satellite images.
+    labels:
+      - cv
+      - segmentation
+      - satellite-images
+```
+
+When you push the resulting file to your git remote it is parsed by Studio and
+it then shows all model artifacts from your `dvc.yaml` files in the model
+registry.
+
+</details>
+
+To get acquainted with how DVC stores and shares data, see our
+[Get Started guide on Data Versioning](/doc/start/data-management/data-versioning).
+
+<details id="push-click-to-see-other-ways-to-add-models">
+
+#### 💡 Expand to see other ways to add models
+
+The other two options are to use the Studio's graphical user interface to add
+models interactively or to manually edit `dvc.yaml` files to add information
+about model artifacts. To get more details on the ways to add models have a look
+at the
+[Model registry documentation](/doc/studio/user-guide/model-registry/add-a-model).
+
+</details>
+
+## DVC Model registry overview
+
+In this guide, we will be using DVC Studio to manage our model registry. DVC
+Studio is a web application which integrates all Iterative tools like DVC and
+adds extra functionality aimed at sharing and collaboration. You can find out
+more about it [here](/doc/studio).
+
+In DVC Studio we can access the model registry by clicking on Models in the top
+bar menu. This will show us a dashboard with all models from all projects we
+have access to.
+
+From the dashboard we will have an overview of all models, latest model versions
+as well stages each of the model versions is assigned to. We can get more
+details for each model by clicking on the model name.
+
+You can check out our
+[example model](https://studio.iterative.ai/team/Iterative/models/b3P4bcYIrGYdzyjqzsf9Xw==/pool-segmentation/v0.1.0)
```

Comment:
@iterative/studio Is this a safe link to use? I'm able to access it when I sign out, but want to make sure it will work for the general public.

Reply:
The link works now but I am not sure about the model ID in the link. If we re-generate the example repo I wonder if it changes and if so how we can make the link resilient to that.

Reply:
Maybe we can link to the more generic https://studio.iterative.ai/team/Iterative/models instead?

Reply:
I guess, but then it won't show the state of the project once this chapter of the tutorial is over.